apps/src/sites/studio/pages/devise/registrations/edit.js

```diff
@@ -24,6 +27,17 @@ const {
 } = scriptData;
 
 $(document).ready(() => {
+  const migrateMultiAuthMountPoint = document.getElementById('migrate-multi-auth');
+  if (migrateMultiAuthMountPoint) {
+    const store = getStore();
+    ReactDOM.render(
+      <Provider store={store}>
```

Comment:
Is a provider/store actually needed in this case?

Reply:
unfortunately, yes, the `Notification` component fails on instantiation if the store isn't provided

Reply:
:man_facepalming: